src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -33,30 +32,30 @@ export function logFileLocation(appName: string) {
 }
 
 const openNewBrowserGameWindow = async (
-  browserUrl: string
+  browserUrl: string,
+  abortController: AbortController
 ): Promise<boolean> => {
+  const mainUrlName = browserUrl.split('://')[1].split('/')[0]
+
   return new Promise((res) => {
     const browserGame = new BrowserWindow({
       icon: icon,
+      fullscreen: true,
       webPreferences: {
+        partition: `persist:${mainUrlName}`,
         webviewTag: true,
         contextIsolation: true,
-        nodeIntegration: true,
-        preload: path.join(__dirname, 'preload.js')
+        nodeIntegration: true
       }
     })
 
-    const url = !app.isPackaged
-      ? 'http://localhost:5173?view=BrowserGame&browserUrl=' +
-        encodeURIComponent(browserUrl)
-      : `file://${path.join(
-          buildDir,
-          './index.html?view=BrowserGame&browserUrl=' +
-            encodeURIComponent(browserUrl)
-        )}`
-
-    browserGame.loadURL(url)
+    browserGame.loadURL(browserUrl)
     setTimeout(() => browserGame.focus(), 200)
```

Comment:
maybe instead of timeout use ready event to avoid race conditions

Reply:
sounds good